docs/tools/vdb_table/README.md

```diff
@@ -70,7 +70,11 @@ tools/
 
 
 Attributes inside vendorX.json has the following properties
-- `support`: Whose values can be `[ "", "none", "partial", "full" ]` indicating on confidence levels, for that attribute support.
+- `support`: Whose values can be `[ "", "none", "partial", "full" ]` indicating on confidence levels, for that attribute support. NOTE: Each change where a "support" claim is being added MUST include a reference to documentation or an example of the functionality being described.
```

Comment:
what about also allowing the addition of links directly to the code that performs the task you are claiming?

Reply:
done